### ISSUE TITLE
Update config.{sub,guess}

### ISSUE THREAD

**Issue by daxtens** [libMesh/libmesh#752]
The versions of config.sub and config.guess are out of date and don't recognise ppc64le. This leads to errors like this:

```
---------------------------------------------
----------- Configuring libMesh -------------
---------------------------------------------
checking build system type... build-aux/config.guess: unable to guess system type

This script, last modified 2012-09-25, has failed to recognize
the operating system you are using. It is advised that you
download the most up to date version of the config scripts from

  http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD
and
  http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD

If the version you run (build-aux/config.guess) is already up to date, please
send the following data and any information you think might be
pertinent to <config-patches@gnu.org> in order to provide the needed
information to handle your system.

config.guess timestamp = 2012-09-25
```

This also affects the bundled version of netcdf.

It can be easily fixed with the following two commands:
```
find . -name config.sub -exec wget -O \{\} 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD' \;
find . -name config.guess -exec wget -O \{\} 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD' \;
```

I've verified that libMesh configures successfully with those files updated.

Thanks in advance! :)

**Comment by jwpeterson** [comment]
> On Dec 15, 2015, at 11:50 PM, Daniel Axtens <notifications@github.com> wrote:
> 
> The versions of configsub and configguess are out of date and don't recognise ppc64le This leads to errors like this:
> 
> 
Thanks for the bug report.  I'll fix in master and 0.9.5 today.

**Comment by jwpeterson** [comment]
Actually, it looks like this is addressed in a more comprehensive way by the automake refresh in #717, so I think I'm just going to merge that as the fix.  It won't be in 0.9.5, but I could be convinced to patch only config.sub and config.guess if someone really wants that.

**Comment by jwpeterson** [comment]
Merged to master in 0d4372d.

**Comment by daxtens** [comment]
Thanks: an autotools refresh will be fine. I'm not particularly worried if it doesn't get fixed immediately: it's a trivial fix on my end. I'm just trying to make sure it percolates up to MOOSE, etc, eventually.
